core/src/main/java/org/infinispan/xsite/GlobalXSiteAdminOperations.java

```diff
@@ -107,6 +108,7 @@ public final String cancelPushState(@Parameter(description = "The destination si
    }
 
    public final Map<String, SiteStatus> globalStatus() {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
```

Comment:
stupid question: isn't `cacheManager.withSubject(Security.getSubject())` required here in this class?

Reply:
They both work. If the `cacheManager` is decorated with a Subject, it will use that, otherwise it uses `Security.getSubject()` which uses a ThreadLocal set by the Security.doAs() call.